barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -115,7 +115,8 @@ transcript. These operations are taken care of by \ref bb::BaseTranscript "Trans
 The Sumcheck output is specified by \ref bb::SumcheckOutput< Flavor >.
  */
 template <typename Flavor> class SumcheckProver {
-
+    // PartiallyEvaluatedMultivariates OR ProverPolynomials
```

Comment:
maybe move this comment above ProverPolynomials and PartiallyEvaluatedMultivariates

Reply:
✅